### ISSUE TITLE
[Screen Readers – Intelligent Support - IRIS]: when focus lands on the buttons screen readers is announcing as the Toggle buttons.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4553]
**PWD Impact:**
Users who rely on screen reader will get impact if screen reader is not announcing the buttons correctly.

**Repro steps:**

Step 1: Open the above URL in a browser. Resource360 Dashboard Page will be displayed. 
Step 2: Navigate to Iris bot icon in the header and invoke it
Step 3: Iris bot chat window will be displayed.
Step 4: Type start over-> Top questions for my role
Step 5: Verify whether the screen reader announces the buttons as toggle buttons or not. 

**Actual Result:**
when we are navigating on the buttons screen readers is announcing as the Toggle buttons.

**Expected Result:**
Screen reader should announce as the buttons only.

**Note:**
Same issue observed in entire chat bot.

**Observation:**
Similar issue observed with NVDA also

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EZBde1C58eFKkdEvHy1sfB0BbfrE-sgqsVO5g2zVY6wbDA?e=jEWXd4

**Test Environment:**

**OS:** Windows 11 22H2 (OS Build 22621.674)
**Browsers:** Edge Chromium (107.0.1418.26)
**Screen Reader:** Narrator
**URL:** https://servicesuat.microsoft.com/#/opdashboard/

**Comment by pgopinadhu** [comment]
#A11yMAS; #Accessibility; #A11ySev2; #Win11-Edge(Chromium); #BotFrameworkWebChat-Jul22; #HCL; #MAS3.3.2

**Comment by compulim** [comment]
@pgopinadhu I couldn't find the Iris bot in the header. The Resource360 Dashboard appears empty to me.

Can you verify if the Iris bot is running?

**Comment by compulim** [comment]
As we cannot continue to investigate without access to the environment and lacking of response from the bug author, I am closing this bug.

**Comment by pgopinadhu** [comment]
@compulim Product Team Provided the Environment access. Please check and let us know if there are any issues. Thanks!

**Comment by compulim** [comment]
I cannot repro this.

When using Windows Narrator, I hear "Top questions for my role, button, off". This indicates the button is a push button and currently not pressed.

You can hear it in the following audio recording.

https://user-images.githubusercontent.com/1622400/224417905-e87b7884-76f3-462d-9e8a-3d61abf9b206.mp4

**Comment by namanimsft** [comment]
![image](https://user-images.githubusercontent.com/66697536/229204851-7caa78de-f969-47dc-bfb5-6ea77466eefc.png)

**Comment by namanimsft** [comment]
@compulim, could you please validate this bug in our test environment https://servicesuat.microsoft.com/#/opdashboard/ and let us know? we are getting the issue and able to reproduce too.

https://user-images.githubusercontent.com/66697536/229205217-8d929443-e3e2-4488-9c0f-6a3eacfc94aa.mp4

**Comment by namanimsft** [comment]
@compulim I'm reopening the bug

**Comment by InduPriya1805** [comment]
@compulim can you please let us know the botframework version and the OS version details
